README.md

```diff
@@ -144,6 +144,40 @@ with **repo** and **workflow** scope and pass it to the action.
 
 To know more about how to pass a secret to GitHub actions you can [Read GitHub Docs](https://docs.github.com/en/actions/reference/encrypted-secrets)
 
+### A note about GitLFS
+
+If your repository uses Git LFS, you will need to manually remove the LFS-related hook files, otherwise the action
```

Comment:
```suggestion
If your repository uses [Git LFS](https://git-lfs.github.com/), you will need to manually remove the LFS-related hook files, otherwise the action
```